openml/runs/run.py

```diff
@@ -155,13 +156,14 @@ def to_filesystem(self, directory: str, store_model: bool = True) -> None:
         run_xml = self._create_description_xml()
         predictions_arff = arff.dumps(self._generate_arff_dict())
 
-        with open(os.path.join(directory, 'description.xml'), 'w') as f:
-            f.write(run_xml)
-        with open(os.path.join(directory, 'predictions.arff'), 'w') as f:
-            f.write(predictions_arff)
+        # It seems like typing does not allow to define the same variable multiple times
+        with open(os.path.join(directory, 'description.xml'), 'w') as fh_1:  # type: TextIO
```

Comment:
Here too, I feel like the context (and one-time use) of the variable does not warrant explicit typing.

Reply:
We discussed offline to keep them for now.